web/src/api/openapi-client/accounts.ts

```
@@ -41,12 +41,14 @@ export type AccountGetQueryResult = NonNullable<
   Awaited<ReturnType<typeof accountGet>>
 >;
 export type AccountGetQueryError =
+  | void
   | UnauthorisedResponse
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)